### PR TITLE
Deleted empty sections + added a line in contributions

### PR DESCRIPTION
**Pull request by MarciaRieferJohnston**
Barrie rocks. Theo too.

### CODE REVIEW

File: docs/guide/writing/style-guides.md [hunk 1]
```diff
@@ -192,13 +192,6 @@ Related resources:
 - [How to Write Good Error Messages – UX Planet](https://uxplanet.org/how-to-write-good-error-messages-858e4551cd4)
 - [How to write better error messages – opensource.com](https://opensource.com/article/17/8/write-effective-error-messages)
 - [How to write better error messages for databases - PostgreSQL](https://www.postgresql.org/docs/current/error-style-guide.html)
-
-### User Guide / Manuals
-
-### Tutorials / How-To
-
-### Concepts, Overview
```
Comment (ericholscher) [inline docs/guide/writing/style-guides.md:200]: 💯  🔥

File: docs/guide/contributing.rst [hunk 2]
```diff
@@ -18,13 +18,15 @@ The guide is meant to address all aspects of software documentation. Depending o
...
+The guide has its origins in the efforts of developers to understand and explain documentation to each other. Since then, Write the Docs has expanded its focus and its community. We've recently reorganized the guide to provide a clearer top-level structure, but there's much work to be done. Feel free to suggest further changes to the organization. Here's where we need your contributions:
 
 * More :ref:`api-documentation` content!
 * Helping non-writer roles understand the moving parts of good docs
 * Helping writers new to developer tools or docs-as-code understand the moving parts of new toolchains and workflows
 * High-level discussions of common tools and toolchains. Please avoid advocacy here! But thoughtful presentation of specific use cases, how you solved them, what worked well, and what didn't work so well, are most welcome. If you want to write this kind of topic, consider starting with an issue in GitHub, or contact one of the new guide editors.
```
Comment (barriebyron) [inline docs/guide/contributing.rst:26]:
```suggestion
* High-level discussions of common tools and toolchains (please avoid advocacy). Present problems, how you solved them, what worked, and what didn't. To write this kind of topic, consider starting with an issue in GitHub or contact one of the guide editors.
```

File: docs/guide/contributing.rst [hunk 3]
```diff
@@ -18,13 +18,15 @@ The guide is meant to address all aspects of software documentation. Depending o
 Where to start
 --------------
 
-The guide has its origins in the efforts of developers to understand and explain documentation to each other. Since then, Write the Docs has expanded its focus and its community. We've recently reorganized the guide to provide a clearer top-level structure, but there's much work to be done. And you should feel free to suggest further changes to the organization, too. Here are the areas we've most recently identified as needing your contributions:
+The guide has its origins in the efforts of developers to understand and explain documentation to each other. Since then, Write the Docs has expanded its focus and its community. We've recently reorganized the guide to provide a clearer top-level structure, but there's much work to be done. Feel free to suggest further changes to the organization. Here's where we need your contributions:
 
 * More :ref:`api-documentation` content!
 * Helping non-writer roles understand the moving parts of good docs
```
Comment (barriebyron) [inline docs/guide/contributing.rst]:
```suggestion
* Helping non-writers understand what makes good documentation
```

File: docs/guide/contributing.rst [hunk 4]
```diff
@@ -18,13 +18,15 @@ The guide is meant to address all aspects of software documentation. Depending o
...
 * Helping writers new to developer tools or docs-as-code understand the moving parts of new toolchains and workflows
```
Comment (barriebyron) [inline docs/guide/contributing.rst]:
```suggestion
* Helping writers new to developer tools or docs-as-code workflows to understand these toolchains and workflows
```

**Review by barriebyron — APPROVED** [review]
thank you all!

**Pull request merged**